scheduler/kubernetes/install.md

```diff
@@ -51,60 +51,37 @@ The native portworx driver in Kubernetes supports the following features:
-| osft | REQUIRED if installing on Openshift.                                                                                          | <var> osft =true</var>                                     |
-| mas    | Specify if PX should run on the Kubernetes master node. For Kubernetes 1.6.4 and prior, this needs to be true (default is false)      | <var>mas=true</var>                                        |
-| z      | Instructs PX to run in zero storage mode on Kubernetes master.                                                                        | <var>z=true</var>                                          |
-| f      | Instructs PX to use any available, unused and unmounted drives or partitions. PX will never use a drive or partition that is mounted. | <var>f=true</var>                                          |
-| st     | Select the secrets type (_aws_, _kvdb_ or _vault_)                                                                                    | <var>st=vault</var>                                        |
-|        | <center>KVDB CONFIGURATION PARAMETERS</center>                                                                                        |                                                            |
-| pwd    | Username and password for ETCD authentication in the form user:password                                                               | <var>pwd=username:password</var>                           |
-| ca     | Location of CA file for ETCD authentication.                                                                                          | <var>ca=/path/to/server.ca</var>                           |
-| cert   | Location of certificate for ETCD authentication.                                                                                      | <var>cert=/path/to/server.crt</var>                        |
-| key    | Location of certificate key for ETCD authentication.                                                                                  | <var>key=/path/to/server.key</var>                         |
-| acl    | ACL token value used for Consul authentication.                                                                                       | <var>acl=398073a8-5091-4d9c-871a-bbbeb030d1f6</var>        |
-|        | <center>LIGHTHOUSE CONFIGURATION PARAMETERS</center>                                                                                  |                                                            |
-| t      | Portworx Lighthouse token for cluster.                                                                                                | <var>t=token-a980f3a8-5091-4d9c-871a-cbbeb030d1e6</var>    |
-| e      | Comma-separated list of environment variables that will be exported to portworx.                                                      | <var>e=API_SERVER=http://lighthouse-new.portworx.com</var> |
-
```

Comment:
I saw that the table got moved into a new page - but, was that wise?

Just want to point out that right now the information is repeated in docs, and the WebSVC -- however we'll move the WebSVC to a wizard (instead of one-page form), so the information would not be duplicated.  This table is relevant for both WebSVC and curl YAML generation.

Reply:
Repetition in docs and WebSVC is true even before this PR. This PR simple moves the table from one page to another.

For people directly using the install.portworx.com HTML form, they don't need to look at the table in the docs site. That was the motive of this PR.